operating/manage-vulnerabilities/vulnerability-management.adoc

```diff
@@ -47,15 +47,61 @@ include::modules/vulnerability-management-review-deferred.adoc[leveloffset=+3]
+
+include::modules/vulnerability-management-scheduled-report.adoc[leveloffset=+2]
 
 .Additional resources
 * xref:../../operating/create-use-collections.adoc#create-use-collections[Creating and using deployment collections]
 * xref:../../operating/create-use-collections.adoc#migrate-scope-collections_create-use-collections[Migration of access scopes to collections]
 * xref:../../integration/integrate-with-email.adoc#configure-acs-kube-security-platform_integrate-with-email[Configuring the email plugin]
 
-include::modules/vulnerability-management-send-report.adoc[leveloffset=+3]
+include::modules/vulnerability-management-send-report.adoc[leveloffset=+2]
+
+include::modules/vulnerability-management-edit-report.adoc[leveloffset=+2]
+
+include::modules/vulnerability-management-delete-report.adoc[leveloffset=+2]
+
+== Vulnerability reporting in Vulnerability Management 2.0
+
+Vulnerability reporting in Vulnerability Management 2.0 provides the ability to create and download an on-demand image vulnerability report. This report includes a comprehensive list of common vulnerabilities and exposures across images and deployments, called workload CVEs in {product-title-short}. You can share this report with auditors or internal stakeholders by scheduling emails in {product-title-short} or by downloading the report and sharing it by using other methods.
+
+:FeatureName: Vulnerability Reporting 2.0
+include::snippets/technology-preview.adoc[]
+
+//creating the report
+include::modules/vulnerability-management20-creating-report.adoc[leveloffset=+2]
+
+//configuring destinations and scheduling
+include::modules/vulnerability-management20-configure-report-delivery-destinations-schedule.adoc[leveloffset=+3]
+
+include::modules/vulnerability-management20-report-review-create.adoc[leveloffset=+3]
 
-include::modules/vulnerability-management-edit-report.adoc[leveloffset=+3]
+//report permissions
+include::modules/vulnerability-management20-permissions.adoc[leveloffset=+2]
 
-include::modules/vulnerability-management-delete-report.adoc[leveloffset=+3]
+//editing reports
+include::modules/vulnerability-management20-edit-reports.adoc[leveloffset=+2]
+
+//downloading reports
+include::modules/vulnerability-management20-download-reports.adoc[leveloffset=+2]
+
+//sending reports
+include::modules/vulnerability-management20-send-reports.adoc[leveloffset=+2]
+
+//cloning reports
+include::modules/vulnerability-management20-clone-reports.adoc[leveloffset=+2]
+
+//deleting reports
+include::modules/vulnerability-management20-delete-reports.adoc[leveloffset=+2]
+
+//report settings
+include::modules/vulnerability-management20-retention-settings.adoc[leveloffset=+2]
+
+
+.Additional resources
```

Comment:
Missing `[role="_additional-resources"]`.

But I also can't tell if this list is meant to apply only to the `vulnerability-management20-retention-settings.adoc` module directly before it, or if it's supposed to be a `=== Additional resources` style (in which case it also needs an ID) to apply to the whole "Vulnerability reporting in Vulnerability Management 2.0" section. The sections referred to in the bullets here aren't actually referenced in the "Configuring vulnerability management report job retention settings" subsection, so I'm guessing it's supposed to apply to the whole 2.0 section.